src/structuralFunctionality/a11yUtils.ts

```diff
@@ -23,7 +25,25 @@ export function setupLiveRegion(params?: LiveRegionParams): HTMLElement {
   }
   return region;
 }
-
+export function setMapDescription(
```

Comment:
Minor: Since we're not setting a value, but rather, getting a value, i think `getMapDescription` makes more sense for the name of the function.